kingpin/actors/aws/elb.py

```diff
@@ -41,6 +41,23 @@
 EXECUTOR = concurrent.futures.ThreadPoolExecutor(10)
 
 
+def dry(dry_message):
+    log.info('Creating _skip_on_dry decorator with "%s"' % dry_message)
+    def _skip_on_dry(function):
+        log.info('Decorating function "%s" with _skip_on_dry' % function)
+        @gen.coroutine
+        def wrapper(self, *args, **kwargs):
+            if self._dry:
+                self.log.info(dry_message)
```

Comment:
pretty big issue that i've discovered here... if you do `dry_message.format(**kwargs)` it will work only when data is passed through *kwargs* and will not work if it's passed through args... meaning...

`yield self._use_certs(elb=found_elb, arn=found_arn)` will work
`yield self._use_certs(found_elb, found_arn)` will not work